sjb/config/test_cases/test_pull_request_ovn_kubernetes_rhel.yml

```diff
@@ -0,0 +1,40 @@
+---
+parent: 'common/test_cases/crio.yml'
+overrides:
+  provision:
+    os: "rhel"
+    stage: "build"
+    provider: "aws"
+  artifacts:
+    - /tmp/openshift-dind-cluster
+    - /tmp/artifacts
+  system_journals:
+    - systemd-journald.service
+extensions:
+  actions:
+  - type: "forward_parameters"
+    parameters:
+      - PULL_NUMBER
+      - PULL_BASE_SHA
+      - PULL_PULL_SHA
+  - type: "script"
+    title: "clone ovn-kubernetes and check out the correct refs"
+    script: |-
+      cd /data/src/github.com/openvswitch/ovn-kubernetes
```

Comment:
you'll probably find this dir is nonexistent -- but this LGTM for testing now, should do generally what you want

Reply:
fixed and squashed. thanks.